src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import auxiliary.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius) {
+        super.setRandomColor();
+        this.radius = radius;
+    }
```

Comment:
```suggestion
    public Circle(int radius, String color) {
        super(color);
        this.radius = radius;
    }
```
do the same for all figure classes